libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -95,23 +105,20 @@ public static synchronized SalesforceAnalyticsManager getInstance(UserAccount ac
     /**
      * Returns the instance of this class associated with this user and community.
      *
-     * @param account     User account.
+     * @param account User account.
      * @param communityId Community ID.
      * @return Instance of this class.
      */
     public static synchronized SalesforceAnalyticsManager getInstance(UserAccount account, String communityId) {
-        if (account == null) {
-            account = SalesforceSDKManager.getInstance().getUserAccountManager().getCurrentUser();
-        }
-        if (account == null) {
-            return null;
-        }
-        String uniqueId = account.getUserId();
-        if (UserAccount.INTERNAL_COMMUNITY_ID.equals(communityId)) {
-            communityId = null;
-        }
-        if (!TextUtils.isEmpty(communityId)) {
-            uniqueId = uniqueId + communityId;
+        String uniqueId = UNAUTH_INSTANCE_KEY;
+        if (account != null) {
```

Comment:
This is a subtle change to the previous behavior.  `getInstance(UserAccount, String)` would previously return the instance associated with the current user.  This changes the behavior to get the unauthenticated instance.  Do you foresee issues with this behavior change?

Reply:
Yeah, I thought about it. We probably shouldn't automatically be substituting `currentUser` anyway. I looked at the callers and everything passes in a user account. I will call it out explicitly in the doc too.